app/models/policies/student_loans/eligibility.rb

```diff
@@ -49,7 +49,7 @@ class Eligibility < ApplicationRecord
       belongs_to :current_school, optional: true, class_name: "School"
 
       validates :claim_school, on: [:"select-claim-school"], presence: {message: ->(object, _data) { object.select_claim_school_presence_error_message }}, unless: :claim_school_somewhere_else?
-      validates :employment_status, on: [:"still-teaching", :submit], presence: {message: ->(object, _data) { "Select if you still work at #{object.claim_school_name}, another school or no longer teach in England" }}
+      validates :employment_status, on: [:submit], presence: {message: ->(object, _data) { "Select if you still work at #{object.claim_school_name}, another school or no longer teach in England" }}
```

Comment:
Question about the existing code - should we have a separate error message for the `logged_in_with_tid_and_has_recent_tps_school?` flow? This error message refers directly to `claim_school_name` rather than tps school...

Reply:
It seems like we should :) I would guess this would be blank in that scenario which we can test.